README.md

```diff
@@ -36,7 +36,32 @@ pacman -S libfdk-aac
 ```
 brew install fdk-aac
 ```
+## Sample usage
 
+```elixir
```

Comment:
A second example for Decoder would be useful too. 
Also, let's add short description what this example does. Something like 
```
The following pipeline takes wav file as input and encodes it as AAC. 
```

Reply:
Done.